test/e2e/precreated_vpc_test.go

```diff
@@ -243,12 +241,6 @@ func createLBService(nsName, svcName, podName string) error {
 	return nil
 }
 
-func getRandomString() string {
-	timestamp := time.Now().UnixNano()
-	hash := sha256.Sum256([]byte(fmt.Sprintf("%d", timestamp)))
-	return hex.EncodeToString(hash[:])[:8]
-}
-
 func (data *TestData) createVPC(orgID, projectID, vpcID string, privateIPs []string, useNSXLB bool) error {
```

Comment:
Could you also add a random suffix for display name? I remember the precreated vpc test may also fail due to DisplayName conflict.
https://github.com/vmware-tanzu/nsx-operator/blob/14457cd6076925a5ebc2129be5ad1dfa0f4d0955/test/e2e/precreated_vpc_test.go#L245-L247

Reply:
ok